### ISSUE TITLE
Unused variable

### ISSUE THREAD

**Issue by justinshenk** [traja-team/traja#65]
https://github.com/traja-team/traja/blob/e3dcf25d69a08e3f58844e96d55bc1ca9917c794/traja/models/visualizer.py#L64

Please remove unused variable `self_connections`, and in the future lint before committing code.

**Comment by Saran-nns** [comment]
Yes, self connections/time-delayed connections are not used in the current latent visualization. Also, note this visualizer support only recurrent networks. At the moment, latent models in traja are feedforward. However, the encoders and decoders have LSTM layer's whose activations/connections can be visualized by passing their hidden to hidden weights to the wrapper.
